### PR TITLE
Add additional Nest SDM API troubleshooting instructions

### PR DESCRIPTION
**Pull request by allenporter**
## Proposed change

Add a link to general url instructions, and Google's documentation
on redirect_uri_mismatch errors.  These seem to be common questions on discord, new issues, or in the community message board.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nest.markdown [hunk 1]
```diff
@@ -191,7 +191,11 @@ script:
 
 ### Troubleshooting
 
-- If you're getting [rickrolled](https://www.youtube.com/watch?v=dQw4w9WgXcQ) instead of being able to see your Nest cameras, you may not have set up your developer account's permissions correctly. Go back through and make sure you've selected read/write under every category that it's an option.
+- See [No URL Available](https://www.home-assistant.io/more-info/no-url-available) for guidance on setup issues related to URLs.
```
Comment (frenck) [inline source/_integrations/nest.markdown]: The more info pages are meant for linking from the UI and will be served to the user when appropriate.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by allenporter** [comment]
Your point about more-info makes sense.

Given all the questions on the forum, folks don't seem to be able to diagnose this problem themselves very well given the existing documentation resources.  External URLs are hard, apparently.   (For myself, for example, I have to use mobile web and connect to my externally exposed port forwarded HA instance because my nat wont loop back.).   Lots of folks just say they see No Url Available and get stuck. 

Is there another general place to stick url trouble shooting instructions for urls? I'm happy to do some legwork given guidance.

**Comment by frenck** [comment]
There is no universal guide on URLs, nor is there a universal answer. There are too many variables and possible setups out there.

File: source/_integrations/nest.markdown [hunk 2]
```diff
@@ -191,7 +191,10 @@ script:
 
 ### Troubleshooting
 
-- If you're getting [rickrolled](https://www.youtube.com/watch?v=dQw4w9WgXcQ) instead of being able to see your Nest cameras, you may not have set up your developer account's permissions correctly. Go back through and make sure you've selected read/write under every category that it's an option.
+
```
Comment (frenck) [inline source/_integrations/nest.markdown]:
```suggestion
```

**Review by frenck — APPROVED** [review]
Thanks, @allenporter 👍

**Pull request merged**